bitnami/oauth2-proxy/Chart.yaml

```diff
@@ -35,4 +35,4 @@ maintainers:
 name: oauth2-proxy
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/oauth2-proxy
-version: 4.1.6
+version: 4.1.7
```

Comment:
I believe this should be a minor bump: you're including a new feature (and even a new template)